Marlin/src/feature/mmu/mmu2.cpp

```diff
@@ -938,6 +960,31 @@ bool MMU2::load_filament_to_nozzle(const uint8_t index) {
     return false;
   }
 
+  #if ENABLED(MMU_IR_UNLOAD_MOVE)
+    // custom THIS IS FOR LOAD TO NOZZLE FUNCTION ONLY
+    if (index != extruder) {
+      if(FILAMENT_PRESENT()) {
+        DEBUG_ECHOLNPGM("Unloading\n");
+        ENABLE_AXIS_E0();
+        filament_ramming(); //unloading from printer side ! EXCLUSIVE FOR THIS FUNCTION
+        while (FILAMENT_PRESENT()) { // if filament is still present, keep unloading - WARNING I'm not sure
+          // if this is a recommended coding style in marlin
+          current_position.e -=1; //this is a guessed value
```

Comment:
Why 1mm here, and 0.25mm in the other location?

Reply:
sorry, 0.25mm is closer to the one check_filament() function, this is just the amount of retraction before checking if any filament is still loaded, there was no reason to make it 1 or .25, 0.25 hypothetically increases the amount of times it has to check, but changing the feedrate by line_to_current_position(MMM_TO_MMS(120)); may play a role for faster or slower filament checks

Reply:
I think for that we would need to depend on your experience with actual hardware to know what works best. 0.25mm seems unnecessarily small, but I'm also not actually trying it with hardware.